traffic_ops/testing/api/v2/topologies_test.go

```diff
@@ -0,0 +1,105 @@
+package v2
```

Comment:
These tests and updates for the following 2 files should be moved to the v3 directory

Reply:
Moved to the v3 directory in ba4d584054.